packages/foundation-legacy/src/ThemeProvider.tsx

```diff
@@ -12,6 +12,8 @@ export interface IThemeProviderProps {
  * for a given scheme name.
  *
  * @param providers - Injected providers for accessing theme data and providing it via a Customizer component.
+ * @deprecated This is an old ThemeProvider implementation. New code should use the ThemeProvider exported from
```

Comment:
Deprecated this old foundation ThemeProvider to reduce potential for confusion when sorting through auto-suggested imports